changelog/add_x86_omf.dd

```diff
@@ -0,0 +1,4 @@
+Added architecture x86_omf for windows-dmd
+
+In windows dmd, an architecture x86_omf is enabled for

```

Comment:
`_On_ windows dmd, an architecture x86_omf is enabled for`

Reply:
how about rewriting the whole sentence a little like
```
On windows dmd the x86_omf architecture can now be used as platform
specification using `--arch=x86_omf`. This is currently equivalent to
the previously existing `--arch=x86` argument.
```

Reply:
That is much better, thanks.